packages/material-ui-lab/src/AlertTitle/AlertTitle.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import clsx from 'clsx';
+
+export const styles = () => ({
+  root: {
+    padding: '0',
+  },
+});
+
+const AlertTitle = React.forwardRef(function AlertTitle(props, ref) {
+  const { classes, children, className } = props;
+  return (
+    <div className={clsx(classes.root, className)} ref={ref}>
+      <Typography component="h2" variant="h6">
```

Comment:
I think that we should merge these two-element, it should render a single DOM node.